client/my-sites/upgrades/checkout/secure-payment-form.jsx

```diff
@@ -62,22 +64,31 @@ const SecurePaymentForm = React.createClass( {
 			return 'free-trial';
 		} else if ( this.state && this.state.userSelectedPaymentBox ) {
 			return this.state.userSelectedPaymentBox;
-		} else if ( cartValues.isCreditCardPaymentsEnabled( cart ) ) {
-			return 'credit-card';
-		} else if ( cartValues.isPayPalExpressEnabled( cart ) ) {
-			return 'paypal';
+		} else if ( cartValues.isPaymentMethodEnabled( cart, preferredPaymentMethods[ 0 ] ) ) {
+			return preferredPaymentMethods[ 0 ];
+		} else if ( cartValues.isPaymentMethodEnabled( cart, preferredPaymentMethods[ 1 ] ) ) {
+			return preferredPaymentMethods[ 1 ];
 		}
 
 		return null;
 	},
 
+	getLocalizedPaymentMethodDefaults( locale ) {
+		const defaults = {
+			de: [ 'paypal', 'credit-card' ],
+			en: [ 'credit-card', 'paypal' ]
```

Comment:
Should we be tying this to language locales? As an example for en, you'll be seeing several different currency types, eg `USD, CAD, AUD, GBP`. I think this payment preference would be more country specific vs language